codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -167,7 +167,7 @@ public void onHttpServerUpgrade(Http2Settings settings) throws Http2Exception {
     @Override
     public void flush(ChannelHandlerContext ctx) throws Http2Exception {
         // Trigger pending writes in the remote flow controller.
-        connection().remote().flowController().writePendingBytes();
+        encoder.flowController().writePendingBytes();
```

Comment:
can you explain what is the difference between the two methods ? Is it basically the same just access the flowconqtroller differently ?